definitions/3.0.0/channelBindingsObject.json

```diff
@@ -84,7 +84,7 @@
     "kafka": {
       "properties": {
         "bindingVersion": {
-          "enum": ["0.4.0", "0.3.0"]
+          "enum": ["v3-0.4.0", "0.4.0", "0.3.0"]
```

Comment:
You still want the other binding versions to be used for v3? 🤔

Reply:
Considering `0.4.0` (or `v3-0.4.0`) is the latest, I think it's ok to still allow the previous one and allow decoupled spec + binding upgrade.